test/specs/jspdf.unit.spec.js

```diff
@@ -898,7 +898,7 @@ describe("Core: Unit Tests", () => {
     doc.__private__.setCustomOutputDestination(writeArray);
     doc.__private__.setLineWidth(595.28);
 
-    expect(writeArray).toEqual(["1687.41 w"]);
+    expect(writeArray).toEqual(doc.__private__.getLineWidth());
```

Comment:
Won't work after changing what the function returns. Just add an assertion like `expect(doc.getLineWidth).toEqual(595.28)`